lib/katello/repository_types/python.rb

```diff
@@ -20,9 +20,16 @@
   publication_class PulpPythonClient::PythonPythonPublication
   publications_api_class PulpPythonClient::PublicationsPypiApi
 
-  generic_remote_option :includes, type: Array, description: "A list containing project specifiers for Python packages to include."
-  generic_remote_option :excludes, type: Array, description: "A list containing project specifiers for Python packages to exclude."
-  generic_remote_option :package_types, type: Array, description: "A list of package types to sync for Python content. Leave blank to get every package type."
+  generic_remote_option :includes, title: "Includes", type: Array, input_type: "textarea", delimiter: "\\n",
+                        description: "Python packages to include from the upstream URL, names separated by newline. You may also specify versions, for example: django~=2.0. Leave empty to include every package."
```

Comment:
To properly handle i18n, we need to mark the title and description as translatable, you can do that with this notation:
```
title: N_("Includes")
```

This records the string as translatable, but returns the untranslated string so it can be translated later using _("") in ruby or other constructs in the UI

Reply:
and actually i think that the UI wouldn't have the needed translations so we'd need to pass it to the UI at render time, which i think we can do